pkg/util/kube/pod.go

```diff
@@ -176,13 +177,15 @@ func GetPodContainerTerminateMessage(pod *corev1api.Pod, container string) strin
 // GetPodTerminateMessage returns the terminate message for all containers of a pod
 func GetPodTerminateMessage(pod *corev1api.Pod) string {
 	message := ""
+	var messageSb179 strings.Builder
 	for _, containerStatus := range pod.Status.ContainerStatuses {
 		if containerStatus.State.Terminated != nil {
 			if containerStatus.State.Terminated.Message != "" {
-				message += containerStatus.State.Terminated.Message + "/"
+				messageSb179.WriteString(containerStatus.State.Terminated.Message + "/")
```

Comment:
Why do we make this change?
The variable name looks weird, is it generated by AI?

Reply:
reverted